### PR TITLE
Don't fail when core API returns HTTP 500 w/o any payload

### PR DESCRIPTION
**Pull request by tisnik**
Just a quick fix not to fail when core API does not return error message.

### CODE REVIEW

File: perf-tests/src/api.py [hunk 1]
```diff
@@ -28,6 +28,9 @@ def get(self):
...
 
     def print_error_response(self, response, message_key):
-        error_message = response.json().get(message_key, "Server does not sent error message")
         print("    Server returned HTTP code {c}".format(c=response.status_code))
-        print("    Error message: {m}".format(m=error_message))
+        try:
+            error_message = response.json().get(message_key, "Server does not sent error message")
+            print("    Error message: {m}".format(m=error_message))
+        except Exception:
+            pass   # no error message
```
Comment (jpopelka) [inline perf-tests/src/api.py]: What do you think about this ?

```python
error_message = None
try:
    error_message = response.json().get(message_key, "Server did not send error message")
except Exception:
    pass

if error_message:
    print("    Error message: {m}".format(m=error_message))

Comment (tisnik) [reply perf-tests/src/api.py]: You are right, let's update the code ;)

**Comment by tisnik** [comment]
[test this]

**Comment by miteshvp** [comment]
@tisnik @jpopelka - will this fix API server 500 error while E2E tests?

**Comment by tisnik** [comment]
@miteshvp nope. This fix is just for performance tests. The issue with server API needs to be fixed in another repo.

**Pull request merged**